ci/infra/testrunner/tests/test_upgrade_from_4_2.py

```diff
@@ -12,6 +12,12 @@ def migrate_node(platform, checker, kubectl, role, node, regcode, option=1):
     platform.ssh_run(role, node, (f'sudo zypper migration --migration {option}'
                                   ' --no-recommends --non-interactive'
                                   ' --auto-agree-with-licenses --allow-vendor-change'))
+
+    # update to latest kubernetes package version, if any
+    k8s_major, k8s_minor, _ =  CURRENT_VERSION.split('.')
```

Comment:
if we are using python 3 we have `from packaging import version` that deals with this on its own instead of manually parsing:

```
>>> from packaging import version
>>> v = version.parse('1.16.5')
>>> v.major
1
>>> v.minor
16
```

Reply:
Good to know, but i think bringing a dependency for one line is maybe overkill.